### ISSUE TITLE
Alternate steam paths

### ISSUE THREAD

**Issue by AdmiringWorm** [MASGAU/MASGAU#58]
Just to inform, the latest beta of steam has now made it available to install games to an alternate steam path:

The only way I have detected to find this is through the registry key for the single game:

```
Key: HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Steam App {steamappnumber}
ValueName: InstallLocation

```

I'll keep looking around to see if the path to the alternative steam library is stored somewhere.

Just something to consider when the latest steam beta is pushed to the public.

**Comment by AdmiringWorm** [comment]
or allow users to add an alternative steam library path, and look for the file "steam.dll" in the selected path.

**Comment by sanmadjack** [comment]
I'll have to investigate this. If worst comes to worst, then steamcommon will just have to use the alt save paths like isntalllocation does. I want to see how this is implemented in final.

**Comment by sanmadjack** [comment]
Hah, forgot to update this! Steam released this feature officially, and soon after I released 1.0.4, which adds support for detecting these paths. In the process I also added the ability to parse steam's config files, which revealed what could be some interesting information. Haha!
